src/services/db.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', '123123', {
```

Comment:
for now it is ok, but in real projects you should put this variables in env

Reply:
That's what we'll do in the next thread ))